analytics/aggregate.go

```diff
@@ -362,6 +363,11 @@ func (f *AnalyticsRecordAggregate) DiscardAggregations(fields []string) {
 			f.OauthEndpoint = make(map[string]map[string]*Counter)
 		case "ApiEndpoint", "apiendpoint":
 			f.ApiEndpoint = make(map[string]*Counter)
+		default:
+			log.WithFields(logrus.Fields{
+				"prefix": MongoAggregatePrefix,
+				"field": field,
+			}).Warning("Invalid field")
```

Comment:
When you see this in the log there's no context for which field is invalid, also not what the action is - can you please add - "Invalid field in the ignore list. skipping"

Reply:
done @letzya !